src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java

```diff
@@ -363,6 +363,13 @@ private void verifyAssetCatalog(Path resourcePath, String platform, String minOS
         }
     }
 
+    /**
+     * Scans the ios/assets folder for possible folders other than
```

Comment:
from this, it is not clear to me if that assets folder is something the user has to supply, or that it is auto-generated.
If the user has to supply it, it should start with src
if it is generated, it should start with gensrc